### ISSUE TITLE
[BUG] Nvidia RTX A5000 (Ampere) GPUs did not be detected

### ISSUE THREAD

**Issue by AlistairMaccallum** [harvester/harvester#6050]
**Describe the bug**
Nvidia RTX A5000 (Ampere) GPUs not detected

**To Reproduce**
- Install Fresh Harvester v1.3.1 single node cluster
- Install pcidevices-controller 
- nvidia-driver-toolkit 
- Supply Nvidia driver download URL (Tested with v470 and v550)

**Expected behavior**
GPUs are detected and added to SR-IOV GPU Devices list

**Support bundle**
Can be provided on demand.

**Environment**
 - Harvester ISO version: 1.3.1
 - Underlying Infrastructure: Baremetal with Asus ESC8000 G4

**Comment by AlistairMaccallum** [comment]
The vGPU appears to be provisioned correctly but the web  VNC viewer only shows the following.
![image](https://github.com/harvester/harvester/assets/81685753/445ea34a-04e4-4b28-8046-7b2f116b0664)

I managed to RDP into the machine eventually and saw the following, in the windows control panel, it looks like the vGPU is detected but not initialised.
![image](https://github.com/harvester/harvester/assets/81685753/0144e452-fceb-454c-8e1d-06ddc0fc27ed)

**Comment by ibrokethecloud** [comment]
the vGPU would need the correct grid drivers. are you able to please confirm that this is installed?

**Comment by AlistairMaccallum** [comment]
@ibrokethecloud yes grid drivers matching the host bundle have been installed on the guest.
